FurEver/Pages/DogQuiz.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@page
+@model FurEver.Pages.DogQuizModel
+@{
+    ViewData["Title"] = "Dog Breed Quiz";
+}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <title>@ViewData["Title"]</title>
+    <link rel="stylesheet" href="~/css/site.css" />
+    <link rel="stylesheet" href="~/css/quiz.css" />
+</head>
+<body>
+    <div class="container">
+        <h1>Dog Breed Quiz</h1>
+        @if (Model.QuizFinished)
+        {
+            <h2>Your Score: @Model.Score / @Model.TotalQuestions</h2>
+            <a href="/DogQuiz" class="btn">Start Again</a>
+            <!-- Link to navigate back to the home page -->
+            <a href="/" class="btn">Back to Home</a> <!-- This is the link to the home page -->
```

Comment:
Remove excessive comments, especially in HTML.
- They will show up in the browser if the user chooses View-Source.
- An href with the label "Back to Home" is a given... the comment does not add any value.